analyses/0012-TUF/analysis.md

```diff
@@ -0,0 +1,533 @@
+- **Contributor documentation:** Analyzes documentation for existing and new
+  contributors to TUF project.
+- **Website:** Analyzes the mechanics of publishing the documentation, including
+  branding, website structure, and maintainability.
+
+Each section begins with summary ratings based on a rubric with appropriate
+[criteria] for the section, then proceeds to:
+
+- **Comments**: Observations about the existing documentation, with a focus on
+  how it does or does not help TUF project users achieve their goals.
+- **Recommendations**: Suggested changes that would improve the effectiveness of
+  the documentation.
+
+The accompanying [implementation](./implementation.md) document breaks the
+recommendations down into concrete actions that can be implemented by project
+contributors. Its focus is on drilling down to specific, achievable work that
+can be completed in constrained blocks of time.
+
+Ultimately, the implementation items are decomposed into a series of
+[issues](./issues.md) that can be implemented by the project maintainers.
+
+### How to use this document
+
+Readers interested only in actionable improvements should skip this document and
+read the **[implementation](./implementation.md) plan** and
+**[issues](./issues.md) list**.
+
+Readers interested in the current state of the documentation and the reasoning
+behind the recommendations should read the section of this document pertaining
+to their area of concern:
+
+- [Project documentation](#project-documentation)
+
+- [Contributor documentation](#contributor-documentation)
+
+- [Website and infastructure](#website-and-infrastructure)
+
+#### Recommendations, requirements, and best practices
+
+This analysis measures documentation against CNCF project maturity standards,
+and suggests possible improvements. In most cases there is more than one way to
+do things. Few recommendations here are meant to be prescriptive. Rather, the
+recommended implementations represent the reviewers' experience with how to
+apply documentation best practices. In other words, borrowing terminology from
+the lexicon of [RFCs][rfc-spec], the changes described here should be understood
+as "recommended" or "should" at the strongest, and "optional" or "may" in many
+cases. Any "must" or "required" actions are clearly denoted as such, and pertain
+to legal requirements such as copyright and licensing issues.
+
+## Project documentation
+
+| Criteria                   | Rating (1-5) |
+| -------------------------- | ------------ |
+| Information architecture   | 2            |
+| New user content           | 1            |
+| Content maintainability    | 3            |
+| Content creation processes | 1            |
+| Inclusive language         | 3            |
+
+Scale:
+
+1 = Not present
+
+2 = Needs improvement
+
+3 = Meets standards
+
+4 = Meets or exceeds standards
+
+5 = Exemplary
+
+### Summary of issues
+
+- Information is repeated is several pages of the website. For example, the TUF
```

Comment:
```suggestion
- Information is repeated on several pages of the website. For example, the TUF
```
I use "on" instead of "in" because, "in" is best suitable when referencing a location of something within another thing. Though they are both prepositions, it's very easy to mix them.